### DISCUSSION TITLE
Venue가 애매한 논문들

### DISCUSSION THREAD

**Discussion by yukyunglee** [yukyunglee/Awesome-Dialogue-State-Tracking#4] [Ideas]
고민되는 Venue 리스트 올려두기

**Comment by Kyumin-Park** [comment]
* *[Domain Adaptive Meta-learning for Dialogue State Tracking](https://ieeexplore.ieee.org/abstract/document/9431715?casa_token=XXvG2B13qhgAAAAA:O-dhl2zV6dYNdLlffscdmJ9Ap6OBR2tYAPcxIWNpx9euy-Qb_1GJcDY7jZy5-o58x02VcHj0)* , TASLP | **DAMAML** | [[Code](https://github.com/DeepLearnXMU/DAMAML)] - IEEE 저널이라 괜찮을것같기도 하네요

**Comment by Kyumin-Park** [comment]
*[Data Augmentation for Copy-Mechanism in Dialogue State Tracking](https://arxiv.org/pdf/2002.09634.pdf)* , ICCS 2021 
*[A Study of Constrained Beam Search for Dialogue State Tracking](https://www.mathieutuli.com/docs/constrained_beam_search.pdf)* , ???
*[Transfer Learning for Unseen Slots in End-to-End Dialogue State Tracking](https://link.springer.com/chapter/10.1007/978-981-15-9323-9_5)* , IWSDS 2019
*[Evaluating Sequence-to-Sequence Modelling for Dialogue State Tracking](https://www.mathieutuli.com/docs/nmt_for_dst.pdf)* , ???

**Reply by yukyunglee** [reply]
[SKIM : Few-Shot Conversational Semantic Parsers with Formal Dialogue Contexts](https://arxiv.org/pdf/2009.07968.pdf), arXiv / Stanford

**Comment by yukyunglee** [comment]
WOZ 

A SIMPLE BUT EFFECTIVE BERT MODEL FOR DIALOG STATE TRACKING ON
RESOURCE-LIMITED SYSTEMS , ???
